contents/dataframes_transform.md

```diff
@@ -16,8 +21,15 @@ In `DataFrames.jl`, the syntax is `source => transformation => target`.
 Like before, we use the `grades_2020` dataset:
 
 ```jl
-@sco process=without_caption_label grades_2020()
+s = "grades_2020()"
+sco(s; process=without_caption_label)
 ```
+<!-- It is not necessary to show the definition of the function here again.
+     This is especially true because the definition of `grades_2021` was
+     not/never shown in the previous section.
+     I used that as an exercise to create it on my own.
+     In case I wouldn't have been able to do that, I would have peeked at
+     the definition of `grades_2020` :-) -->
```

Comment:
```suggestion
```
Lengthy comments like this can be put in GitHub issue/PR so that they don't end up in the file.

Reply:
I know that this *would* be possible. Unfortunately than my work flow would be much more complicated, because I make these comments directly when reading. The comments I would have to add *after*
- I have read the chapter/section,
- made the commits, and
- made the PR

So please forgive me when I'll continue adding comments to the code.  
(The other option would be to *not* add these comments at all, but then I guess some of the stuff that I suggest would not be self-explanatory ...)